sktime/forecasting/arch/_uarch.py

```diff
@@ -0,0 +1,362 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+"""Implements Generalized Autoregressive Conditional Heteroskedasticity models."""
+
+__author__ = ["Vasudeva-bit"]
+__all__ = ["ARCH"]
+
+import pandas as pd
+
+from sktime.forecasting.base import BaseForecaster
+from sktime.utils.validation._dependencies import _check_soft_dependencies
+
+
+class ARCH(BaseForecaster):
+    r"""Directly interfaces ARCH models from python package arch.
+
+    ARCH models are a popular class of volatility models that use observed values of
+    returns or residuals as volatility shocks to forecast the volatility in high
+    frequency time series data..
+
+    A complete ARCH model is divided into three components:
+        a mean model, e.g., a constant mean or an ARX;
+        a volatility process, e.g., a GARCH or an EGARCH process; and
+        a distribution for the standardized residuals.
+
+    Parameters
+    ----------
+    mean : str, optional
+        Name of the mean model.  Currently supported options are: 'Constant',
+        'Zero', 'LS', 'AR', 'ARX', 'HAR' and  'HARX'
+    lags : int or list (int), optional
+        Either a scalar integer value indicating lag length or a list of
+        integers specifying lag locations.
+    vol : str, optional
+        Name of the volatility model.  Currently supported options are:
+        'GARCH' (default), 'ARCH', 'EGARCH', 'FIARCH' and 'HARCH'
+    p : int, optional
+        Lag order of the symmetric innovation
+    o : int, optional
+        Lag order of the asymmetric innovation
+    q : int, optional
+        Lag order of lagged volatility or equivalent
+    power : float, optional
+        Power to use with GARCH and related models
+    dist : int, optional
+        Name of the error distribution.  Currently supported options are:
+            * Normal: 'normal', 'gaussian' (default)
```

Comment:
Is this case insensitive? The test configuration has it as `Normal`?

Reply:
Will check that